packages/dmn-editor/tests/e2e/mutations/deleteNode.spec.ts

```diff
@@ -0,0 +1,228 @@
+
+      await expect(nodes.get({ name: DefaultNodeName.DECISION })).not.toBeAttached();
+    });
+
+    test("Text Annotation", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.TEXT_ANNOTATION, targetPosition: { x: 100, y: 100 } });
+      await nodes.delete({
+        name: DefaultNodeName.TEXT_ANNOTATION,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.TEXT_ANNOTATION })).not.toBeAttached();
+    });
+
+    test("Decision Service", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.DECISION_SERVICE, targetPosition: { x: 100, y: 100 } });
+      await nodes.delete({
+        name: DefaultNodeName.DECISION_SERVICE,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.DECISION_SERVICE })).not.toBeAttached();
+    });
+
+    test("BKM", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.BKM, targetPosition: { x: 100, y: 100 } });
+      await nodes.delete({
+        name: DefaultNodeName.BKM,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.BKM })).not.toBeAttached();
+    });
+
+    test("Knowledge Source", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.KNOWLEDGE_SOURCE, targetPosition: { x: 100, y: 100 } });
+      await nodes.delete({
+        name: DefaultNodeName.KNOWLEDGE_SOURCE,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.KNOWLEDGE_SOURCE })).not.toBeAttached();
+    });
+  });
+
+  test.describe("Container", () => {
+    test("Group", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.GROUP, targetPosition: { x: 100, y: 100 } });
+      await palette.dragNewNode({ type: NodeType.DECISION, targetPosition: { x: 130, y: 130 } });
+
+      await expect(nodes.get({ name: DefaultNodeName.GROUP })).toBeAttached();
+      await expect(nodes.get({ name: DefaultNodeName.DECISION })).toBeAttached();
+
+      await nodes.delete({
+        name: DefaultNodeName.GROUP,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.GROUP })).not.toBeAttached();
+      await expect(nodes.get({ name: DefaultNodeName.DECISION })).toBeAttached();
+    });
+
+    test("Decision Service", async ({ diagram, palette, nodes }) => {
+      await palette.dragNewNode({ type: NodeType.DECISION_SERVICE, targetPosition: { x: 100, y: 100 } });
+      await palette.dragNewNode({ type: NodeType.DECISION, targetPosition: { x: 130, y: 130 } });
+
+      await expect(nodes.get({ name: DefaultNodeName.DECISION_SERVICE })).toBeAttached();
+      await expect(nodes.get({ name: DefaultNodeName.DECISION })).toBeAttached();
+
+      await nodes.delete({
+        name: DefaultNodeName.DECISION_SERVICE,
+      });
+
+      await expect(nodes.get({ name: DefaultNodeName.DECISION_SERVICE })).not.toBeAttached();
+      await expect(nodes.get({ name: DefaultNodeName.DECISION })).toBeAttached();
+    });
```

Comment:
@tiagobento this is the only scenario I would like to double-check, as there was huge discussion around decision services in the past. I struggle to find that ticket. Is it expected deleting a Decision Service node that few other nodes, causes deletion only of this decision service container and other nodes are kept?